rudder-agent/SOURCES/configure

```diff
@@ -212,23 +212,38 @@ fi
 test_cc() {
   TEST_FILE=$(mktemp /tmp/cc.XXXXXX)
   echo "void main() {}" > "${TEST_FILE}.c"
-  if cc -o "$2" "$3" "${TEST_FILE}" "${TEST_FILE}.c" 2>/dev/null
```

Comment:
did not work as expected with several options (but still returned zero so it was always enabled):

```
+ cc -o -fstack-protector '-Wl,-z,relro -Wl,-z,now' /tmp/cc.N2CQ1s /tmp/cc.N2CQ1s.c
/usr/bin/ld: warning: -z relro -Wl ignored
```